elastio-ec2-backup_ecs_stopped_task/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1
+
+# Version elastio cli what will use like base image.
+# Most likely this is not the latest version of elastio cli.
+# You can find the latest version at this link.
+# https://gallery.ecr.aws/elastio-dev/elastio-cli
+FROM public.ecr.aws/elastio-dev/elastio-cli:0.17.0-master
```

Comment:
Do *NOT* ever hard-code a `master` version of the CLI.  We never want customers running anything but release bits.  Check with @Hexawolf to confirm, but I think you can safely use `:latest` here to always get the latest CLI.